packages/connectivity/src/scp-cf/jwt.spec.ts

```diff
@@ -252,37 +252,37 @@ describe('jwt', () => {
     });
   });
 
-  describe('audiences', () => {
+  describe('audiences()', () => {
     it('returns a set of the entries of the "aud" claim, if present. If a dot is present, we only take everything before the dot', () => {
       const token = {
         aud: ['one', 'two.one', 'three.two.one']
       };
 
-      expect(audiences(token)).toEqual(new Set(['one', 'two', 'three']));
+      expect(audiences(token)).toEqual(['one', 'two', 'three']);
     });
 
     it('returns an empty set if the "aud" claim is empty', () => {
       const tokenEmpty = { aud: [] };
 
-      expect(audiences(tokenEmpty)).toEqual(new Set());
+      expect(audiences(tokenEmpty)).toEqual([]);
     });
 
-    it('returns a set of the entries of the "scope" claim iff the "aud" claim is not present and if the entry contains a dot (and then we again only take everything before the dot', () => {
+    it('returns audiences from scope, if no "aud" property exists', () => {
       const token = {
         scope: ['one', 'two.one', 'three.two.one']
       };
 
-      expect(audiences(token)).toEqual(new Set(['two', 'three']));
+      expect(audiences(token)).toEqual(['two', 'three']);
```

Comment:
Not sure why 'one' is not included

Reply:
Because it only takes those that are in front of the "."